mu4/domain/notation/internal/notationplayback.h

```diff
@@ -35,10 +36,19 @@ class NotationPlayback : public INotationPlayback, public async::Asyncable
 public:
     NotationPlayback(IGetScore* getScore);
 
+    void init();
+
     std::shared_ptr<audio::midi::MidiStream> midiStream() const override;
 
-    QRect playbackCursorRectBySec(float sec) const override;
-    QRect playbackCursorRectByTick(uint32_t tick) const override;
+    float tickToSec(int tick) const override;
+    int secToTick(float sec) const override;
+
+    QRect playbackCursorRectByTick(int tick) const override;
+
+    RetVal<int> playPositionTick() const override;
+    void setPlayPositionTick(int tick) override;
+    bool setPlayPositionByElement(const Element* e) override;
```

Comment:
can be called as setPlayPositionTick(const Element * e) as is the case with setPlayPositionTick(int tick)?
I think that the BySome suffix is not needed since we can look at the receiving parameter.

Reply:
`setPlayPositionTick` sets the tick and that's it.
`setPlayPositionByElement` sets the tick which is getting by means of the element